src/renderers/shared/fiber/ReactFiberScheduler.js

```diff
@@ -171,40 +166,47 @@ module.exports = function<T, P, I, TI, C>(config : HostConfig<T, P, I, TI, C>) {
     // ref unmounts.
     let effectfulFiber = finishedWork.firstEffect;
     while (effectfulFiber) {
-      switch (effectfulFiber.effectTag) {
-        case Placement:
-        case PlacementAndCallback: {
-          commitPlacement(effectfulFiber);
-          // Clear the "placement" from effect tag so that we know that this is inserted, before
-          // any life-cycles like componentDidMount gets called.
-          effectfulFiber.effectTag ^= Placement;
-          break;
-        }
-        case PlacementAndUpdate:
-        case PlacementAndUpdateAndCallback: {
-          // Placement
-          commitPlacement(effectfulFiber);
-          // Clear the "placement" from effect tag so that we know that this is inserted, before
-          // any life-cycles like componentDidMount gets called.
-          effectfulFiber.effectTag ^= Placement;
-
-          // Update
-          const current = effectfulFiber.alternate;
-          commitWork(current, effectfulFiber);
-          break;
+      try {
```

Comment:
Is my latest commit okay? Had to add another loop outside so that we can continue after an error